pkgs/applications/office/kitsas/default.nix

```
@@ -0,0 +1,47 @@
+{ lib, mkDerivation, fetchFromGitHub, qmake, qtsvg, qtcreator, libsForQt5, libzip, pkg-config  }:
+
+mkDerivation rec {
+  pname = "kitsas";
+  version = "2.3";
+
+  src = fetchFromGitHub {
+    owner = "artoh";
+    repo = "kitupiikki";
+    rev = "v" + version;
+    sha256 = "1qac6cxkb45rs5pschsf2rvpa789g27shmrwpshwahqzhw42xvgl";
+  };
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ qmake qtsvg libsForQt5.poppler libzip ];
+
+  phases = [ "unpackPhase" "configurePhase" "buildPhase" "installPhase" "fixupPhase" ];
+
+  # We use a separate build-dir as otherwise ld seems to get confused between
+  # directory and executable name on buildPhase.
+  configurePhase = ''
+    mkdir build-linux
+    cd build-linux
+    qmake $src/kitsas/kitsas.pro -spec linux-g++ "CONFIG+=release" && make qmake_all
+  '';
+  preFixup = ''
+    make clean
+    rm Makefile
+  '';
+  installPhase = ''
+    mkdir -p $out/bin $out/share
+    cp kitsas $out/bin
+    cp $src/kitsas.png $out/share
+    cp $src/kitsas.desktop $out/share
+    chmod 644 $out/share/kitsas.desktop
+  '';
+
+  enableParallelBuilding = true;
+
+  meta = with lib; {
+    homepage = "https://github.com/artoh/kitupiikki";
+    description = "An accounting tool suitable for Finnish assocications and small business";
+    maintainers = [ "gspia" ];
```

```suggestion
    maintainers = with maintainers; [ gspia ];
```
Also please add yourself to the maintainers list in a separate commit.

---

Added gspia into maintainers in one commit and after it made the mainainers-line change in another commit.